src/config/users/docker.properties

```diff
@@ -1,3 +1,4 @@
 app_url=http://orderlyweb:8080
+app.port=8081
```

Comment:
This should probably follow underscore convention as in the other keys

Reply:
In fact this must not be working at the moment!